mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,353 @@
+import numpy as np
+from .base import _get_final_est, BaseEstimator, _check_estimator
+
+
+class ReceptiveField(BaseEstimator):
```

Comment:
We already have `rf.tmin`, `rf.tmax`, `rf.sfreq`, so it should be `rf.times` and `rf.delays` or all of these should have the trailing `_`.  @kingjr?